proto/wasmrollapp/cron/v1beta1/params.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package wasmrollapp.cron.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/dymensionxyz/rollapp-wasm/x/cron/types";
+
+// Params defines the set of module parameters.
+message Params {
+  // Security address that can whitelist/delist contract
+  repeated string security_address = 1 [    
+    (gogoproto.jsontag) = "security_address,omitempty",
+    (gogoproto.moretags) = "yaml:\"security_address\""
+  ];
+  // set module enabled or not
+  bool enable_cron = 2;
```

Comment:
by adding the `enabled` field to `CronJob` I believe we can get rid of this one

Reply:
removed